src/components/SearchInput/index.jsx

```diff
@@ -1,19 +1,66 @@
-import { useRef, useState } from "react";
+import { useRef, useState, useEffect } from "react";
+import axios from "axios";
+import useUserInputStore from "../../store/store";
 
 function SearchInput() {
-  const [query, setQuery] = useState("");
+  const { userInput, setUserInput } = useUserInputStore();
+  const [autoCompletions, setAutoCompletions] = useState([]);
+  const [selectedItemIndex, setSelectedItemIndex] = useState(-1);
   const inputRef = useRef(null);
 
-  function handleQueryChange(event) {
-    setQuery(event.target.value);
+  function handleUserInputChange(event) {
+    setUserInput(event.target.value);
   }
 
-  function handleSubmit(event) {
-    event.preventDefault();
+  useEffect(() => {
+    async function getAutoCompletions(userInput) {
+      const baseURL = "http://localhost:3000";
```

Comment:
```suggestion
      const baseURL = import.meta.env.VITE_BASE_URL;
```
저는 .env를 이용했는데 통일 시키면 좋을 것 같아요!

Reply:
네! 수정하겠습니다!